chore: minor updates (#3369)
JetBrains/kotlin-web-site: docs/topics/gradle/gradle-compilation-and-caches.md

### Limit of custom values

To collect build scans' statistics, Kotlin build reports use [Gradle's custom values](https://docs.gradle.com/enterprise/tutorials/extending-build-scans/). 

> Different Gradle plugins and yourself can also write data to custom values. The number of custom values has a limit.

Reviewer: ```suggestion
Both you and different Gradle plugins can write data to custom values. The number of custom values has a limit.
```
